tests/test-surround-view.cpp

```diff
@@ -69,16 +70,16 @@ struct SVOutConfig {
 
 #if HAVE_GLES
 
-static void dump_dma_video_buf (SmartPtr<VideoBuffer> buf, const char *prefix_name, uint32_t idx)
+static void dump_dma_video_buf (SmartPtr<VideoBuffer> buf, const char *prefix_name)
 {
     char file_name[256];
     XCAM_ASSERT (prefix_name);
     XCAM_ASSERT (buf.ptr ());
 
     const VideoBufferInfo &info = buf->get_video_info ();
     snprintf (
-        file_name, 256, "%s-%dx%d.%05d.%s.yuv",
```

Comment:
to line 85: 
inside in function tex->dump_texture_image (file_name);
modify fopen (..., 'wb+')